test/e2e/cypress/integration/profiles/verification/login/success.spec.js

```diff
@@ -0,0 +1,37 @@
+import {
+  APP_URL,
+  assertVerifiableAddress,
+  gen,
+  parseHtml,
+  verifyHrefPattern,
+} from '../../../../helpers'
+
+context('Verify', () => {
+  describe('success flow', () => {
+    let identity
+    before(() => {
+      cy.deleteMail()
+    })
+
+    beforeEach(() => {
+      cy.dontLoginUserAfterRegistration()
+      cy.enableLoginForVerifiedAddressOnly()
+
+      identity = gen.identity()
+      cy.register(identity)
+    })
+
```

Comment:
These tests need a `afterEach()` or `after()` which resets `cy.dontLoginUserAfterRegistration` and `cy.enableLoginForVerifiedAddressOnly` to the previous values or all subsequent tests will use this set up and thus fail.